### PR TITLE
added community token mape

### PR DESCRIPTION
**Pull request by pepebndc**

### CODE REVIEW

File: networks/meld/tokens/0xC89D6b70DcBD4DDf9e5E474Bb529d3F49Ef1c653.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "CMCslug": "",
+    "decimals": 18,
```
Comment (thuyvu17) [inline networks/meld/tokens/0xC89D6b70DcBD4DDf9e5E474Bb529d3F49Ef1c653.json:3]:
```suggestion
    "decimals": 8,
```

File: networks/meld/tokens/0xC89D6b70DcBD4DDf9e5E474Bb529d3F49Ef1c653.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "mldId": "meld-mape-0xC89D6b70DcBD4DDf9e5E474Bb529d3F49Ef1c653",
+    "orgName": "MAPE",
+    "orgUrl": "",
+    "tokenAddress": "0xC89D6b70DcBD4DDf9e5E474Bb529d3F49Ef1c653",
+    "tokenColorTheme": 1,
+    "tokenIcon": "/static/tokens/mape.svg",
```
Comment (infrmtcs) [inline networks/meld/tokens/0xC89D6b70DcBD4DDf9e5E474Bb529d3F49Ef1c653.json:9]: Minor thing: Should we capitalize the icon file name for consistency?

File: networks/meld/tokens/0xC89D6b70DcBD4DDf9e5E474Bb529d3F49Ef1c653.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    "tokenName": "Mape",
```
Comment (infrmtcs) [inline networks/meld/tokens/0xC89D6b70DcBD4DDf9e5E474Bb529d3F49Ef1c653.json:10]: Based on explorer data:
```suggestion
    "tokenName": "mape",
```

File: networks/meld/tokens/0xC89D6b70DcBD4DDf9e5E474Bb529d3F49Ef1c653.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
+{
+    "CMCslug": "",
+    "decimals": 18,
+    "mldId": "meld-mape-0xC89D6b70DcBD4DDf9e5E474Bb529d3F49Ef1c653",
+    "orgName": "MAPE",
+    "orgUrl": "",
+    "tokenAddress": "0xC89D6b70DcBD4DDf9e5E474Bb529d3F49Ef1c653",
+    "tokenColorTheme": 1,
```
Comment (infrmtcs) [inline networks/meld/tokens/0xC89D6b70DcBD4DDf9e5E474Bb529d3F49Ef1c653.json:8]: Color 1 is red, while this icon looks more blue to me.

Comment (pepebndc) [reply networks/meld/tokens/0xC89D6b70DcBD4DDf9e5E474Bb529d3F49Ef1c653.json:8]: this was requested by the creator... I also find it weird, but let's keep it red

File: networks/meld/tokens/0xC89D6b70DcBD4DDf9e5E474Bb529d3F49Ef1c653.json (hunk 2 above)
Comment (pepebndc) [reply networks/meld/tokens/0xC89D6b70DcBD4DDf9e5E474Bb529d3F49Ef1c653.json]: done

**Review by infrmtcs — APPROVED** [review]

**Pull request merged**